### ISSUE TITLE
Gift criteria error

### ISSUE THREAD

**Issue by szabolcstari** [ankitpokhrel/WooCommerce-Multiple-Free-Gift#17]
Hello!

I installed WooCommerce Multiple Free Gif, but when I do a shopping, it does not take notice the criterias,. When I set greater then 2, it gives the gift even if I just choose 1 item. If I use "equal' it does the same, if I try to use cart total criterias, it does not work also.. Pls help anyway the customers will take all the products for free...... this is not working at all, sorry..

**Comment by szabolcstari** [comment]
Hello! Did you get any similar issue so far?

**Comment by ankitpokhrel** [comment]
No! Can you send snapshot of criteria you are using?

**Comment by szabolcstari** [comment]
Hello!
I used these 2 criterias. What it should do is: if someone buy 2 (or more,
so at least 2) then he/she can get 1 for free. I tried to use the number of
items in the cart, and tried to use prices as you see as a by pass way..
Thank you!



Szabolcs TARI
mobil: +36 30 627 8087
Ad Futurus Foundation [ adfuturus.hu | info@adfuturus.hu | 1066. Oktogon 3,
1sst floor - apt. 1
Derigo Travel Ltd. | derigo.me | info@derigo.me | 1066. Oktogon 3, 1sst
floor - apt. 1
Facebook: https://www.facebook.com/szabolcs.tari
Linkedin: https://www.linkedin.com/in/szabolcstari

2016-09-01 8:56 GMT+02:00 Ankit Pokhrel <notifications@github.com>:
